.github/workflows/build.yml

```diff
@@ -33,14 +31,6 @@ jobs:
             cp "$RUNNER_TEMP/LICENSE" "$DIR/age"
             go build -o "$DIR/age" -ldflags "-X main.Version=$VERSION" -trimpath ./cmd/...
             if [ "$GOOS" == "windows" ]; then
-              for exe in "$DIR"/age/*.exe; do
```

Comment:
Thank you for working on this!

We could also just add an if around this loop that checks if secrets.SIGN_PASS is empty?

Reply:
I can do that, but it creates an edgecase. If the secret ever gets removed it would silently fail to sign.

Reply:
Which do you prefer?

Reply:
That's a valid concern, but I'd rather keep it simple. Failing to sign a binary is not the end of the world. (I would feel differently about the risk of leaking the key, for example.)